app/repository/family.py

```diff
@@ -179,19 +181,19 @@ def search(db: Session, query_params: dict[str, str]) -> list[FamilyReadDTO]:
         search.append(Family.description.ilike(term))
 
     if "geography" in query_params.keys():
-        term = query_params["geography"]
+        term = cast(str, query_params["geography"])
```

Comment:
how did we get away without these casts before :man_shrugging: ... nice tidy :smile: